src/solvers/configEnvVars.m

```diff
@@ -0,0 +1,56 @@
+function configEnvVars(printLevel)
+
+    global GUROBI_PATH;
+    global ILOG_CPLEX_PATH;
+    global TOMLAB_PATH;
+    global MOSEK_PATH;
+    global ENV_VARS;
+
+    if nargin < 1
+        printLevel = 0;
+    end
+
+    if ENV_VARS.STATUS == 0
+        solverPaths = {};
+        solverPaths{1,1} = 'ILOG_CPLEX_PATH';
+        solverPaths{1,2} = {'/Applications/IBM/ILOG/CPLEX_Studio1262', '/Applications/IBM/ILOG/CPLEX_Studio1263', '/Applications/IBM/ILOG/CPLEX_Studio127', ...
+                            '/opt/ibm/ILOG/CPLEX_Studio1262', '/opt/ibm/ILOG/CPLEX_Studio1263', '/opt/ibm/ILOG/CPLEX_Studio127', ...
+                            'C:\Program Files\IBM\ILOG\CPLEX_Studio1262', 'C:\Program Files\IBM\ILOG\CPLEX_Studio1263', 'C:\Program Files\IBM\ILOG\CPLEX_Studio127'};
+        solverPaths{2,1} = 'GUROBI_PATH';
+        solverPaths{2,2} = {'/Library/gurobi600', '/Library/gurobi650', '/Library/gurobi702', '/opt/gurobi650', '/opt/gurobi70', 'C:\gurobi600', 'C:\gurobi650', 'C:\gurobi70'};
+        solverPaths{3,1} = 'TOMLAB_PATH';
+        solverPaths{3,2} = {'/opt/tomlab', 'C:\tomlab', '/Applications/tomlab'};
+        solverPaths{4,1} = 'MOSEK_PATH';
+        solverPaths{4,2} = {'/opt/mosek/7/', '/opt/mosek/8/', '/Applications/mosek/7', '/Applications/mosek/8', 'C:\Program Files\Mosek\7', 'C:\Program Files\Mosek\8'};
+
```

Comment:
The thought is good - the implementation is too system dependent - maybe later.

The path is set to the `path` as a user would have set by following the instructions in `SOLVERS.md`. The user has to set the environment variable when installing the solver - this is purely for convenience if the default paths have been used in the past when installing a solver, but the environment variables have not been set (e.g. this is the case for existing users).